lib/pact/init/cli.rb

```diff
@@ -0,0 +1,25 @@
+require 'thor'
+require 'pact/init'
+
+module Pact
+  module Init
+    class CLI < Thor
+
+      desc 'create' , "run the command line interface"
+      method_option :consumer, aliases: "--consumer", :desc => "The name of your consumer"
+      method_option :provider, aliases: "--provider", :desc => "The name of your provider"
+
+      def create(args = {})
+        if !options['consumer'].nil?
+          args[:consumer] = options['consumer']
```

Comment:
You could use the more idiomatic Ruby here, `if options['consumer']`, with the same result I believe.